server/repository/src/test_db.rs

```diff
@@ -59,9 +59,15 @@ pub async fn setup(db_settings: &DatabaseSettings) -> StorageConnectionManager {
 
     let connection_manager =
         ConnectionManager::<DBBackendConnection>::new(&db_settings.connection_string());
+    const SQLITE_LOCKWAIT_MS: u32 = 5000; //5 second wait for test lock timeout
     let pool = Pool::builder()
-        //.max_size(1)
+        // .max_size(1)
```

Comment:
is this needed ?

Reply:
No, will remove, defaults to 10, but we could explicitly state something?

Reply:
tbh only noticed since a space was added